MetaMask/metamask-extension
# fix: Prevent NFT banner from tracking more than once

## **Description**

This PR prevents the NFT banner from being tracked more than once per render.

[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/23774?quickstart=1)

## **Related issues**

Fixes: N/A

## **Manual testing steps**

1. Add a console.log to the NFT tab's `useEffect` on an account with no NFTs
2. Only see the log happen once.

## **Screenshots/Recordings**



### **Before**

N/A

### **After**

N/A

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
